src/moves/MultiParticleBrownianMotion.h

```diff
@@ -525,7 +527,7 @@ inline void MultiParticleBrownian::RotateForceBiased(uint molIndex)
 
   matrix = RotationMatrix::FromAxisAngle(rotLen, rot.Normalize());
 
-  XYZ center = newCOMs.Get(molIndex);
```

Comment:
I cannot make a comment in Line 464, but similar to Multiparticle, we need to use `omp critical` for `XYZ MultiParticleBrownian::CalcRandomTransform(XYZ const &lb, double const max, uint molIndex)` function.

Reply:
Yes. I will do that if I can't find a work-around to needing the critical section with the MultiParticle move.